src/ldp.rs

```diff
@@ -61,6 +71,34 @@ pub trait ProofSuite {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct ProofPreparation {
+    pub proof: Proof,
+    pub jws_header: Header,
+    pub signing_input: Vec<u8>,
+}
+
+impl ProofPreparation {
```

Comment:
Great, just wanted to note as we discussed in our call earlier that a future work item might be to allow interpretation of `signing_input` data through OOB indicators to potentially allow a client to display a neutral interface to the user of what is to be signed exactly, and any further implications according to trust frameworks.

Reply:
I will open an issue.